packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -170,7 +170,8 @@ class Dart2JSTarget extends Dart2WebTarget {
     final String platformBinariesPath = artifacts.getHostArtifact(HostArtifact.webPlatformKernelFolder).path;
     final List<String> sharedCommandOptions = <String>[
       artifacts.getArtifactPath(Artifact.engineDartBinary, platform: TargetPlatform.web_javascript),
-      artifacts.getArtifactPath(Artifact.dart2jsSnapshot, platform: TargetPlatform.web_javascript),
```

Comment:
Do we still need `Artifact.dart2jsSnapshot` for anything? If not, can you please also delete the enum value?

Reply:
I have removed all references to the dart2js and dart2wasm snapshots.

Reply:
PTAL